### PR TITLE
Update self hosted login prompt

### PR DESCRIPTION
**Pull request by elibud**
**Fixes** #8636 

**To test:**

From a clean install trigger LogIn and check that the screen matches the design.

simulator screen shot - iphone 7 - 2018-02-20 at 08 03 52

CC: @folletto

### CODE REVIEW

**Comment by elibud** [comment]
There's one thing pending which is hiding the buttons on the IPhone SE when the keyboard is displayed, I'm wondering if you are already handling similar cases somewhere else in the flow @ScoutHarris @nheagy? Thanks

simulator screen shot - iphone se - 2018-02-20 at 08 30 50

**Comment by etoledom** [comment]
There is also this issue:  With bigger font, the buttons are not displayed completely. 

We will start working on these details in the Accessibility project, but this screen seems to be specially challenging in terms of the use of space.

simulator screen shot - iphone 5 - 2018-02-20 at 08 32 00

**Comment by elibud** [comment]
Yep, with the previous design this screen looked bad on bigger font sizes and it has not improved with the current one. As part of the Accessibility project we will need to produce guidelines so we future designs and implementations adjust properly.

simulator screen shot - iphone se - 2018-02-20 at 08 58 39

File: WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift [hunk 1]
```diff
@@ -5,20 +5,22 @@ import GoogleSignIn
...
     @IBOutlet var emailTextField: WPWalkthroughTextField!
-    @IBOutlet var selfHostedSigninButton: UIButton!
     @IBOutlet var bottomContentConstraint: NSLayoutConstraint?
     @IBOutlet var verticalCenterConstraint: NSLayoutConstraint?
     @IBOutlet var inputStack: UIStackView?
-    @objc var onePasswordButton: UIButton!
-    @objc var googleLoginButton: UIButton?
+    @IBOutlet var alternativeLoginLabel: UILabel?
+
+    var onePasswordButton: UIButton!
```
Comment (nheagy) [inline WordPress/Classes/ViewRelated/NUX/LoginEmailViewController.swift]: losing those `@objc` is wonderful

**Pull request merged**